modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectDefinitionLocalServiceTest.java

```diff
@@ -2965,6 +3115,24 @@ private ObjectDefinition _addCustomObjectDefinition(
 		return _addCustomObjectDefinition(className, name, name, name);
 	}
 
+	private ObjectDefinition _addCustomObjectDefinition(
+			String name, String scope,
+			List<ObjectDefinitionSetting> objectDefinitionSettings)
+		throws Exception {
+
+		return _objectDefinitionLocalService.addCustomObjectDefinition(
+			TestPropsValues.getUserId(), 0, null, false, false, true, false,
+			false, LocalizedMapUtil.getLocalizedMap(name), name, null, null,
+			LocalizedMapUtil.getLocalizedMap(name), true, scope,
+			ObjectDefinitionConstants.STORAGE_TYPE_DEFAULT,
+			objectDefinitionSettings,
+			Arrays.asList(
+				ObjectFieldUtil.createObjectField(
+					ObjectFieldConstants.BUSINESS_TYPE_TEXT,
+					ObjectFieldConstants.DB_TYPE_STRING,
+					RandomTestUtil.randomString(), StringUtil.randomId())));
+	}
+
```

Comment:
Could you check if there’s an existing method in `ObjectDefinitionTestUtil` that can be used instead of creating a new one?

Reply:
There is not, since we're adding the `objectDefinitionSettings` now.